config/initializers/raven.rb

```diff
@@ -1,4 +1,6 @@
 Raven.configure do |config|
   config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
   config.environments = %w(staging production)
+  config.excluded_exceptions += ['OpenSSL::SSL::SSLError']
```

Comment:
```suggestion
  config.excluded_exceptions += ['OpenSSL::SSL::SSLError', 'Mastodon::UnexpectedResponseError', 'HTTP::TimeoutError', 'HTTP::ConnectionError']
```

この辺もかなり多いので加えて欲しい :pray:

Reply:
👍 👍 👍